server/server.go

```diff
@@ -79,12 +80,45 @@ var dirPaths = []string{
 	"trie",
 }
 
+// set log output to the designated file
+func setLogFileWriter(logFilePath string) (*os.File, error) {
+	// if file path is empty string return error
+	if logFilePath == "" {
+		return nil, fmt.Errorf("log file path not defined")
+	}
+
+	logFile, err := os.Create(logFilePath)
+	if err != nil {
+		return nil, fmt.Errorf("could not create log file, %w", err)
+	}
+
+	return logFile, nil
+}
+
+// create new logger; log to file if log path is defined
```

Comment:
Comment should be in the form of:
`// newLogger creates a new logger...`

Reply:
Fixed fe41b1e3945aa3ff7c35ff6095786ce8d717692c